### ISSUE TITLE
Crash when resuming activity

### ISSUE THREAD

**Issue by Gebatzens** [Etuldan/spaRSS#128]
This crash happens sometimes when I touch the notification or launcher icon.
I use cm 12.1 on a Nexus 5 with spaRSS 1.10.5

```
E/AndroidRuntime(12162): java.lang.RuntimeException: Unable to resume activity {net.etuldan.sparss/net.etuldan.sparss.activity.HomeActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void net.etuldan.sparss.adapter.DrawerAdapter.a(int)' on a null object reference
E/AndroidRuntime(12162): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3019)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3050)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2425)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.access$900(ActivityThread.java:154)
E/AndroidRuntime(12162): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
E/AndroidRuntime(12162): 	at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(12162): 	at android.os.Looper.loop(Looper.java:135)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.main(ActivityThread.java:5294)
E/AndroidRuntime(12162): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(12162): 	at java.lang.reflect.Method.invoke(Method.java:372)
E/AndroidRuntime(12162): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
E/AndroidRuntime(12162): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
E/AndroidRuntime(12162): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void net.etuldan.sparss.adapter.DrawerAdapter.a(int)' on a null object reference
E/AndroidRuntime(12162): 	at net.etuldan.sparss.activity.HomeActivity.b(SourceFile:322)
E/AndroidRuntime(12162): 	at net.etuldan.sparss.activity.HomeActivity.onNewIntent(SourceFile:233)
E/AndroidRuntime(12162): 	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1211)
E/AndroidRuntime(12162): 	at android.app.Instrumentation.callActivityOnNewIntent(Instrumentation.java:1223)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.deliverNewIntents(ActivityThread.java:2489)
E/AndroidRuntime(12162): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3001)
E/AndroidRuntime(12162): 	... 11 more
```

**Comment by nehrwert** [comment]
I have the same problem. spaRSS crashes in ~ 90 percent of the cases when I touch the notification and in ~20 percent when i touch launcher icon. I use Android 5.0 in the Motrorolla flavour. I don't know where the logs are.

**Comment by Urban82** [comment]
This crash was fixed with commit 9fc4f7b0d82b6615c94954a3715765e3b3213aa7. This issue can be closed.
